Geometry/HcalCommonData/BuildFile.xml

```diff
@@ -1,6 +1,7 @@
 <use   name="DetectorDescription/Base"/>
 <use   name="DetectorDescription/Core"/>
 <use   name="DataFormats/HcalDetId"/>
+<use   name="SimDataFormats/CaloTest"/>
```

Comment:
In general, CommonData should be used in both reco and sim. Please, check that you do not bring in PCaloHit sim hit dependency for reco.